### PR TITLE
Prepare 2.0.0 release

### PR DESCRIPTION
**Pull request by klappradla**
Updated the Changelog and the version number accordingly. Did not yet add a tag.

Am I missing something 🤔

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,8 +15,35 @@ https://github.com/bitcrowd/rubocop-bitcrowd/compare/v1.3.0...HEAD
 ### Potentially breaking changes:
 
 * Put potentially breaking changes here (in a brief bullet point)
+
+### New features:
+
+* Put new features here (in a brief bullet point)
```
Comment (nickgnd) [inline CHANGELOG.md]: Just a minor remark.

Maybe using the _italic_ style can emphasize that this point is not a new feature, what do you think?

File: CHANGELOG.md [hunk 2]
```diff
@@ -15,8 +15,35 @@ https://github.com/bitcrowd/rubocop-bitcrowd/compare/v1.3.0...HEAD
...
+
+### Fixes:
+
+* Put fixes here (in a brief bullet point)
```
Comment (nickgnd) [inline CHANGELOG.md]: The same here.

File: CHANGELOG.md [hunk 3]
```diff
@@ -15,8 +15,35 @@ https://github.com/bitcrowd/rubocop-bitcrowd/compare/v1.3.0...HEAD
...
+We in addition to the default directories, e.g. want to exclude `log`, `tmp` and `storage` in Rails projects.
+
+In order to keep excluding both, the bitcrowd patterns and the RuboCop's default ones, add this to your `.rubocop.yml` file:
+
+```yml
+inherit_mode:
+  merge:
+    - Exclude
+```
+
```
Comment (nickgnd) [inline CHANGELOG.md:42]: 👍

**Review by nickgnd — APPROVED** [review]
Thanks a lot for all your work!  👏 🏅 🙇

File: CHANGELOG.md (hunk 1 above)
Comment (klappradla) [reply CHANGELOG.md]: Nice idea 💡

**Review by tessi — APPROVED** [review]
🎉

**Comment by klappradla** [comment]
Hm, now wondering, is `2.0.0` the right version number 🤔 

My reasoning was: it's a breaking change since people need to adapt their config in order to get the previous behavior back. However, it's not really a "backwards incompatible" breaking change (if that's even possible with this library) - it's more of a deprecation. People could e.g. also continue to fully override `AllCops:Exclude`.

So maybe we need to go with `1.4.0` ❓ 

Reading this ☝️ I'm actually tending more towards the latter 🙈

**Pull request merged**